example/flaskr/templates/recipe/view.html

```diff
@@ -19,30 +22,41 @@ <h1>{{ recipe['title'] }}</h1>
         {% endif %}
     </header>
     <p class="body">{{ recipe['body'] }}</p>
-    <ul>
+    <table>
+        <thead>
+            <tr>
+                <th>Ingredient</th>
+                <th>Amount</th>
+                <th>Image</th>
+                <th>Pros</th>
+                <th>Cons</th>
+                <th>Other Choices</th>
+
+            </tr>
+        </thead>
         {% for ingredient in ingredients %}
-        <li>
-            <h2>{{ingredient['name']}}, {{ ingredient['amount'] }} {{ ingredient['unit']}} </h2>
-            You
-            <ul>
-                <li> Nourishment: {{ingredient['r_nourishment']}}</li>
-                <li> Value: {{ingredient['r_value']}}</li>
-            </ul>
-            Others
-            <ul>
-                <li> Human Welfare: {{ingredient['r_human_welfare']}}</li>
-                <li> Animal Welfare: {{ingredient['r_animal_welfare']}}</li>
-            </ul>
-            Planet
-            <ul>                
-                <li> Resource Conservation: {{ingredient['r_resource_cons']}}</li>
-                <li> Biodiversity: {{ingredient['r_biodiversity']}}</li>
-                <li> Global Warming: {{ingredient['r_global_warming']}}</li>
-            </ul>
-            <br><br>
-        </li>
+        <tr>
+            <td>{{ ingredient['name'] }}</td>
+            <td>{{ ingredient['amount'] }} {{ ingredient['unit'] }}</td>
+            <td> IMAGE PLACEHOLDER </td>
+            <td class="ingredient-pros">
+                {% for rating in ratings %}
+                    {% if ingredient[rating] >= goodCutoff %}
+                    <span class="good-rating"> {{rating}}: {{ ingredient[rating] }}</span>
+                    {% endif %}
+                {% endfor %}
+            </td>
+            <td class="ingredient-cons">
+                {% for rating in ratings %}
+                    {% if ingredient[rating] <= badCutoff %}
```

Comment:
I think you will run into problems if both this and the `goodCutoff` check for equality. Maybe change this one to simply be less than?

Reply:
Done! Thanks for catching that.